api/v1alpha1/etcdcluster_webhook.go

```diff
@@ -106,3 +121,66 @@ func (r *EtcdCluster) ValidateDelete() (admission.Warnings, error) {
 	etcdclusterlog.Info("validate delete", "name", r.Name)
 	return nil, nil
 }
+
+// validatePdb validates PDB fields
+func (r *EtcdCluster) validatePdb() (admission.Warnings, field.ErrorList) {
+	if r.Spec.PodDisruptionBudget == nil {
+		return nil, nil
+	}
+	pdb := r.Spec.PodDisruptionBudget
+	var warnings admission.Warnings
+	var allErrors field.ErrorList
+	if pdb.Spec.MinAvailable != nil && pdb.Spec.MaxUnavailable != nil {
```

Comment:
Can we implement this with kubebuilder annotations? We don't need to support code if controller runtime already has capabilities to do it. Or maybe we can use CEL for such purpose

Reply:
There's no kubebuilder marker for `one of`